src/pages/open-forge-app/open-forge-app.tsx

```diff
@@ -40,6 +39,8 @@ export class OpenForgeApp {
           <stencil-route url="/contact" component="app-contact" />
           <stencil-route url="/opportunities" component="app-opportunities" />
           <stencil-route url="/about" component="app-about" />
+          <stencil-route url="/404" component="app-404" exact={true} />
```

Comment:
Yep good comment - I had that there intentionally because I think right now we should redirect everyone to the home page, since our 404 page isn't a "sell" page.

What do you think is a better experience?   Taking them to the 404, or taking them to home?

Reply:
404 page is for the scenario something goes wrong, so if something went wrong the expected flow is redirect to 404 instead of home.

So maybe we should see which scenarios are good to redirect home or 404

Reply:
The 404 page looks good, If the Code review passes we can merge in and I can recheck on the dev URL. 
@Fdom92  I see what you're saying, But the time we put in to identify the good flows we could just use to fix the issue. This is just a temporary patch to keep it from looking broken